src/platform/user/profile/vet360/tests/helpers/addressValidationMessages.unit.spec.js

```diff
@@ -0,0 +1,193 @@
+import { expect } from 'chai';
+import {
+  getValidationMessageKey,
+  showAddressValidationModal,
+} from '../../../utilities';
+
+describe('getValidationMessageKey', () => {
+  it('returns showSuggestionsOverride key', () => {
+    const addressValidationError = false;
+    const validationKey = 12345;
+    const suggestedAddresses = [
+      {
+        address: {
+          addressLine1: '400 N 65th St',
+          addressType: 'DOMESTIC',
+          city: 'Seattle',
+          countryName: 'USA',
+          countryCodeIso3: 'USA',
+          countyCode: '53033',
+          countyName: 'King',
+          stateCode: 'WA',
+          zipCode: '98103',
+          zipCodeSuffix: '5252',
+        },
+        addressMetaData: {
+          confidenceScore: 79.0,
+          addressType: 'Domestic',
+          deliveryPointValidation: 'UNDELIVERABLE',
+        },
+      },
+      {
+        address: {
+          addressLine1: '400 NE 65th St',
+          addressType: 'DOMESTIC',
+          city: 'Seattle',
+          countryName: 'USA',
+          countryCodeIso3: 'USA',
+          countyCode: '53033',
+          countyName: 'King',
+          stateCode: 'WA',
+          zipCode: '98115',
+          zipCodeSuffix: '6463',
+        },
+        addressMetaData: {
+          confidenceScore: 98.0,
+          addressType: 'Domestic',
+          deliveryPointValidation:
+            'STREET_NUMBER_VALIDATED_BUT_MISSING_UNIT_NUMBER',
+          residentialDeliveryIndicator: 'RESIDENTIAL',
+        },
+      },
+    ];
+    expect(
+      getValidationMessageKey(
+        suggestedAddresses,
+        validationKey,
+        addressValidationError,
+      ),
+    ).to.equal('showSuggestionsOverride');
```

Comment:
I never know if we should use the constants in tests. Would love to hear if others have strong opinions on this.

Reply:
@erikphansen this isn't using a constant - its testing a helper method that returns a key that relates to the messages.  I prefer to leave this test in place incase a request to change the message copy / logic is presented.

Reply:
Yeah I made this comment in error. I thought that `'showSuggestionsOverride'` was also stored in a constant, but it's actually the _key_ on the constants object.